HeartRateMonitor.py

```diff
@@ -0,0 +1,21 @@
+class HRM:
+    avghr = -1
+    insthr = []
+    times_insthr = []
+    diagnosis = ''
+    times_diagnosis = []
+
+    def __init__(self, times=[], voltages=[], time_units='s'):
+        self.times = times
+        self.voltages = voltages
+
+    def avghr(self, t1=-1, t2=-1):
+        from AvgHR import avghr
+        if t1 == -1 or t2== -1:
+            t1 = self.times[0]
+            t2 = self.times[len(self.times)-1]
+        return avghr(self.times, self.voltages, t1, t2)
+
+    def ihr(self):
+        from InstHR import ihr
+        return ihr(self.times, self.voltages)
```

Comment:
I would try to convert `ihr` to a native method; this module import construct is a bit clunky (though totally due to the way that we are doing this assignment).